src/index.html

```diff
@@ -7,16 +7,64 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+      http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css">
+
   </head>
+
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#"><img src="images/logo.png" alt="MoYo"></a>
```

Comment:
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
code styling recommendations. check cases below as well
```suggestion
        <a href="#">
          <img src="images/logo.png" alt="MoYo">
        </a>
```

Reply:
Adicionei a solução proposta